proxy_config

```diff
@@ -0,0 +1,9 @@
+port 8080;
```

Comment:
Nothing wrong at all, but our config for server also use port 8080. So we will change it to 8081 or something when we use it.